### PR TITLE
Fixing cypress tests for Angular Test Application

### LINKED ISSUE #1696: Simple storage api for micro frontends
Luigi client should have a function set that provides similar functionality to html web storage because of potential problem with 3rd party cookie problems. Since it requires a communication with the core, they must be async.
On luigi core side, the received data from the client are managed by storing in localstorage scope of the main app, under a distinct key that allows mapping to a specific micro frontend (e.g. domain/location/url of the micro frontend). Implementation should be in a way, that the storage strategy can be changed or made customizable later.

Example:
`
LuigiClient.storageManager().setItem('myKey', value).then( () => { console.log('value stored successfully'); });
`
`
LuigiClient.storageManager().getItem('myKey').then( value => { doSomething(value); });
`

### PR DESCRIPTION
**Pull request by legteodav**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**
fixes #1696

### CODE REVIEW

File: test/e2e-test-application/src/luigi-config/extended/helpers.js [hunk 1]
```diff
@@ -117,6 +125,8 @@ export const projectsCounterFn = context =>
 export const projectsNavProviderFn = context =>
   new Promise(resolve => {
     getAllProjects().then(function(result) {
+      console.log('Inside projectsNavProviderFn --> ', result);
```
Comment (UlianaMunich) [inline test/e2e-test-application/src/luigi-config/extended/helpers.js:128]: clean up :)

File: test/e2e-test-application/src/luigi-config/extended/navigation.js [hunk 2]
```diff
@@ -301,38 +302,19 @@ class Navigation {
...
-      });
-    }
-    if (projectExists('pr4')) {
-      items.push({
-        icon:
-          'https://pbs.twimg.com/profile_images/1143452953858183170/QLk-HGmK_bigger.png',
-        label: 'Project 4',
-        link: '/projects/pr4'
+
+    console.log('getProductSwitcherItems allProjects =>', getAllProjectsSync());
```
Comment (UlianaMunich) [inline test/e2e-test-application/src/luigi-config/extended/navigation.js:306]: clean up :)

**Review by UlianaMunich — CHANGES_REQUESTED** [review]
Tests working like a charm locally, just a tiny cleanup left

**Review by UlianaMunich — APPROVED** [review]

**Review by ndricimrr — APPROVED** [review]
👍

**Pull request merged**